Include/internal/pycore_object.h

```diff
@@ -94,6 +94,15 @@ PyAPI_FUNC(void) _Py_NO_RETURN _Py_FatalRefcountErrorFunc(
 #define _Py_FatalRefcountError(message) \
     _Py_FatalRefcountErrorFunc(__func__, (message))
 
+#define _PyReftracerTrack(obj, operation) \
+    do { \
+        struct _reftracer_runtime_state *tracer = &_PyRuntime.ref_tracer; \
+        if (tracer->tracer_func != NULL) { \
+            void* data = tracer->tracer_data; \
+            tracer->tracer_func(obj, (operation), data); \
+        } \
+    } while(0)
+ 
```

Comment:
The linter wants to kill trailing spaces:

```suggestion
    } while(0)

```